json2xml/utils.py

```diff
@@ -27,7 +27,7 @@ def readfromjson(filename: str) -> dict[str, str]:
     Reads a json string and emits json string
     """
     try:
-        json_data = open(filename)
+        json_data = open(filename,encoding="UTF-8")
```

Comment:
Please add a space after the comma, else our CI won't pass.

Reply:
It was my first time to be tested with ci.
I thought a person would do it until I did a pull request. 
I found out that it was an automatic test after doing the pull request, 
but I was careless because there was no grammatical error. 
Thank you so much for giving me this opportunity(An opportunity to add a portion of the code to the known library) to have such a good experience.

Reply:
@XEL-Maker No issues. Yes, we have automated tests that run everytime you commit a change to ensure nothing breaks.

You are always welcome to contribute to the library and I would be more than happy to review changes.